run.sh

```diff
@@ -346,6 +349,25 @@ check_plugins() {
   done <<< "$(WP plugin list --field=name)"
 }
 
+check_packages() {
+  # If a composer.json file exists in /app => install it
+  if [[ -f "/app/composer.json" ]]; then
+    h3 "Installing packages listed in composer.json"
+    composer install
+    return
+  fi
+
+  if [[ "${REQUIRE-}" ]]; then
```

Comment:
Installing volumed composer.json and then adding packages from REQUIRE is totally feasible as well.